### DISCUSSION TITLE
Is there a list of cookies used in Opencart?

### DISCUSSION THREAD

**Discussion by kanenas** [opencart/opencart#13773] [Q&A]
Because of **Google Consent Mode v2**, I am trying to make a list of the cookies used in Opencart.
So far I have found these:
- **OCSESSID**: This cookie is essential for maintaining your session on the website. It helps keep you logged in and preserves your shopping cart contents.
- **language**: Stores your preferred language for the website, ensuring that the content is displayed in the correct language.
- **currency**: Stores your preferred currency, ensuring that prices are displayed in the currency you prefer.

Does anyone has to add some more?

**Comment by ADDCreative** [comment]
A good start would be to do a search for setcookie. Don't forget localstorage and sessionstorage.
https://github.com/search?q=repo%3Aopencart%2Fopencart+setcookie&type=code
https://github.com/search?q=repo%3Aopencart%2Fopencart+localstorage&type=code
https://github.com/search?q=repo%3Aopencart%2Fopencart+sessionstorage&type=code
